app/config/config_dev.yml

```diff
@@ -5,6 +5,12 @@ framework:
     router:   { resource: "%kernel.root_dir%/config/routing_dev.yml" }
     profiler: { only_exceptions: false }
 
+doctrine:
+    dbal:
+        driver: pdo_sqlite
+        path: %kernel.root_dir%/Resources/sqlite.db
```

Comment:
It's a work in progress. I'm sorry I did not denote that more clearly. Adding the path to `.gitignore` is trivial. But I'd prefer to determine a suitable location before doing that.